apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -406,11 +406,11 @@ export const renderListingApps = () => (
 export const renderAllowManager = ({
   modelId,
   type,
-  wording,
+  isRenameRequest,
 }: {
   modelId: DeviceModelId;
   type: Theme["theme"];
-  wording: string;
+  isRenameRequest?: boolean;
```

Comment:
Not mandatory at all, but maybe a union of string literals (enum) could be more flexible ? Like `requestType: "renaming" | "manager"` with a default to `"manager"` ? WDYT ?

Reply:
I'm good with your proposal, it's more open to changes. I will do it, Thanks